.gitignore

```diff
@@ -1,4 +1,5 @@
 .vs
+.vscode
```

Comment:
I think we decided to version control this, since there's configuration in there to setup the linter within the IDE?

Could be thinking of layouts, but we'd want to do that on both eventually. @ryanpetersonOF?

Might be that we want to ignore the folder, but still include certain files?..

Reply:
Yeah we had a 'in the real world' discussion about this.  It should be removed from the .gitignore in favo(u)r of eslint configs